api/src/main/java/org/pmiops/workbench/utils/FieldValues.java

```diff
@@ -12,13 +12,13 @@
 import java.util.List;
 import java.util.Optional;
 import java.util.stream.Collectors;
-import org.jetbrains.annotations.NotNull;
+import javax.annotation.Nonnull;
 import org.joda.time.DateTime;
 
 /** Utility class for working with FieldValueLists, FieldValues, and Fields */
 public final class FieldValues {
 
-  public static final int MICROSECONDS_IN_MILLISECOND = 1000;
+  private static final int MICROSECONDS_IN_MILLISECOND = 1000;
```

Comment:
my intention here is that BQ clients don't need to be explicitly aware that they need to make this conversion